### PR TITLE
Add twingate_group resource - UPD

### PR DESCRIPTION
**Pull request by vmanilo**
Original PR https://github.com/Twingate/terraform-provider-twingate/pull/56

### CODE REVIEW

**Review by romankor — CHANGES_REQUESTED** [review]
I went over all the comments from the previous PR and everything seems to be addressed 

here is another comment that made on the other PR 
https://github.com/Twingate/terraform-provider-twingate/pull/56/files#r851820228

and also a small on on that one

File: twingate/client_group_test.go [hunk 1]
```diff
@@ -189,12 +295,13 @@ func TestClientGroupReadRequestError(t *testing.T) {
 		group, err := client.readGroup(context.Background(), groupId)
 
 		assert.Nil(t, group)
-		assert.EqualError(t, err, "failed to read group with id id: Post \""+client.GraphqlServerURL+"\": error_1")
+		assert.EqualError(t, err, fmt.Sprintf(`failed to read group with id id: Post "%s": error_1`, client.GraphqlServerURL))
```
[the remark this answers is not in the document]
Comment (vmanilo) [reply twingate/client_group_test.go]: yes, updated groupIds in other tests as well

**Comment by vmanilo** [comment]
added terraform example to create/import twingate_group

**Review by romankor — APPROVED** [review]
GTM

**Pull request merged**